corehq/apps/cleanup/management/commands/delete_doc_conflicts.py

```diff
@@ -9,18 +10,18 @@ def bulk_delete(db, docs):
     if not docs:
         return
 
-    print("Deleting {} doc revisions".format(len(docs)))
+    logging.info("Deleting {} doc revisions".format(len(docs)))
```

Comment:
Do I need to pick logging handler e.g. logging.getLogger() or is the main one fine here? Is there a general policy?